packages/sitecore-jss-nextjs/tsconfig.json

```diff
@@ -23,6 +23,7 @@
     "allowSyntheticDefaultImports": true,
     "esModuleInterop": true,
     "resolveJsonModule": true,
+    "useUnknownInCatchVariables": false,
```

Comment:
Just curious - what was the reason for adding this?

Reply:
Build for the nextjs package was failing in the editing-render-middleware.ts file.
I got this error in the catch block. 
![image](https://user-images.githubusercontent.com/89087450/152551909-af922236-d018-4469-870e-cb949f9c53d2.png)
Starting typescript v4.0, catch clause variables have type unknown instead of any.
This is a workaround I found.

Reply:
Hmm. This change actually seems to be breaking the build on CI.

`tsconfig.json(26,5): error TS5023: Unknown compiler option 'useUnknownInCatchVariables'.`

We should probably revert this. Guessing it's something on your local env that's causing the error for you?

Reply:
Yeah, probably something with my local environment.